### DISCUSSION TITLE
What name for my i18n crate ?

### DISCUSSION THREAD

**Discussion by Baptistemontan** [leptos-rs/leptos#1572] [General]
I'm currently working on a lib for easy translation in leptos: https://github.com/Baptistemontan/leptos_i18n.
I'm quite happy of where it is now and would like to publish it.

But I don't want to use a name that the leptos team would like to use in the future, my crate is more about translations than full i18n supports so I'm thinking about `leptos_translate`, would that be ok to do ?

**Comment by gbj** [comment]
`leptos_translate` sounds great! I'm very happy with people using `leptos_` for crate names in the ecosystem, and I have no plans to publish a similarly-named i18n crate, so go for it! Feel free to make a PR to add it to https://github.com/leptos-rs/awesome-leptos/ for visibility when it's ready.

**Reply by Baptistemontan** [reply]
Thanks for the replie! I'll wait for v0.5.x to be stabilized before publishing it because it will be a merge hell if done before.

Thanks for the PR idea, I'll notify once published.
